app/javascript/components/enrollment/Enrollment.js

```diff
@@ -184,17 +187,28 @@ class Enrollment extends React.Component {
         this.setState({ index: lastIndex, lastIndex: null });
```

Comment:
same here

Reply:
fixed in https://github.com/SaraAlert/SaraAlert/pull/975/commits/ed5b1fa40df633f1cbe6d4556e87ba0f7ae8f545